.github/workflows/test_linux_packages.yml

```diff
@@ -0,0 +1,68 @@
+# Copyright 2025 Advanced Micro Devices, Inc.
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+name: Test Linux Packages
+
+on:
+  workflow_dispatch:
+    inputs:
+      package_version:
+        type: string
+        default: ADHOCBUILD
+
+  workflow_call:
+    inputs:
+      package_version:
+        type: string
+        default: ADHOCBUILD
+
+jobs:
+  test_rocm_info:
+    name: "Sanity ROCM Test"
+    runs-on: linux-mi300-1gpu-ossci
+    strategy:
+      fail-fast: true
+    defaults:
+      run:
+        shell: bash
+    env:
+      VENV_DIR: ${{ github.workspace }}/.venv
+
+    # MOSTLY BOILER PLATE ABOVE.
+    steps:
+      - name: Install the AWS tool
+        run: sudo apt-get install -y awscli
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@ececac1a45f3b08a01d2dd070d28d111c5fe6722 # v4.1.0
+        with:
+          aws-region: us-east-2
+          role-to-assume: arn:aws:iam::692859939525:role/therock-artifacts
+
```

Comment:
```suggestion
```
No need to configure credentials as anonymous read access is configured for files in the bucket.